common/changes/@uifabric/styling/master_2017-08-31-16-34.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
+      "packageName": "@uifabric/styling",
+      "type": "none"
```

Comment:
Could you update the comment and type here? There is a change to the Styling package.